docs/concepts/storage/persistent-volumes.md

```diff
@@ -109,6 +109,39 @@ However, the particular path specified in the custom recycler pod template in th
 
 For volume plugins that support the Delete reclaim policy, deletion removes both the `PersistentVolume` object from Kubernetes, as well as deleting the associated storage asset in the external infrastructure, such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume. Volumes that were dynamically provisioned inherit the [reclaim policy of their `StorageClass`](#reclaim-policy-1), which defaults to Delete. The administrator should configure the `StorageClass` according to users' expectations, otherwise the PV must be edited or patched after it is created. See [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
 
+
+### Expanding Persistent Volumes Claims
```

Comment:
Please consider moving this to the `## PersistentVolumeClaims` section.
This article has never touched the concept of `PersistentVolumeClaim` before that section.

Reply:
hmm, I disagree. Resizing is part of volume's lifecycle and concept of claim has already been introduced. For example from Reclaiming text that appears just above resizing doc:

"When a user is done with their volume, they can delete the PVC objects from the API which allows reclamation of the resource"

So surely users will know about persistent volume claims at this point.

Reply:
The intent of this Alpha feature is about resizing/expanding persistent volumes, but it has to be done via an edit to the corresponding PersistentVolumeClaim. I still suggest we move it down to the PVC subsection so that the logic in this topic flows. But I may be wrong, so I won't insist.